guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to different needs. These configurations include **Federated Mints**, **Federated Single Guardian Mints**, and **Solo Mints (Unfederated Mints)**, each with its strengths and trade-offs.
+
+## Role of Guardians
+Guardians are participants who collaboratively manage a federations operations. They secure the federation's funds using multi-sig wallets, validate transactions through consensus, and control ecash issuance and redemption.
+
+## Role of Gateways
```

Comment:
On the intro page, I mentioned that the illustration about connectivity using lighting felt out of place. It seems to fit well with this part about gateways. Isn't the gateway stuff generally a thing and maybe should not be in the fedimint page?

Reply:
‘Gateway’ is a term used only for Fedimint. It’s unique because it’s a separate service that a federation needs to set up in order to be lightning compatible. Cashu doesn’t use ‘gateway.’  To connect a lightning node on Cashu the user simply edits a mint setup file and plugs in some mint info, like an admin macaroon and TLS cert (at least for LND—other implementations may require different details). Since gateways aren’t relevant for Cashu, I didn’t include them in the introduction.

I agree that this illustration is great for visualizing the role of the gateway. I’ll modify it to show only Fedimint federations so it’s clearer that it’s a Fedimint-specific service enabling lightning and connecting federations to the lightning network.